core/backend/Process/LegacyHandler/PortalUserActivator.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Process\LegacyHandler;
```

Comment:
Hi @eojedapilchik this file is missing the copyright

Reply:
added